### PR TITLE
Update mp.md

### PR DESCRIPTION
**Pull request by matentzn**

### CODE REVIEW

**Review by jamesaoverton — APPROVED** [review]
Looks good to me.

**Comment by matentzn** [comment]
@sbello should provide more accurate information

**Comment by sbello** [comment]
Checks are failing 
FAILURES:
ERROR:ontology/mp.md: 47:1: wrong indentation: expected 2 but found 0 (indentation)
ERROR:ontology/mp.md: 50:4: wrong indentation: expected 4 but found 3 (indentation)
make: *** [Makefile:138: extract-metadata] Error 1

Not sure where this is. I was going to add more users but will hold off for now. I'm guessing when Nico did this he missed a bit of formatting. Or do I not need to worry that the OBOFoundryTest is failing?

File: ontology/mp.md [hunk 1]
```diff
@@ -43,6 +43,12 @@ browsers:
   - label: Monarch
     title: Monarch Phenotype Page
     url: http://monarchinitiative.org/phenotype/MP:0000001
+usages:
+  - user: http://www.informatics.jax.org/vocab/mp_ontology
+    description: MGI uses MP for phenotype annotation
```
Comment (matentzn) [inline ontology/mp.md:48]: @sbello provide better description?

File: ontology/mp.md [hunk 2]
```diff
@@ -43,6 +43,12 @@ browsers:
...
+    examples:
+      - url: http://www.informatics.jax.org/vocab/mp_ontology/MP:0005375
+        description: Adipose phenotype annotations
```
Comment (matentzn) [inline ontology/mp.md:51]: @sbello provide better description of what kind of information one can access at that URL related to the MP term?

File: ontology/mp.md [hunk 3]
```diff
@@ -43,6 +43,12 @@ browsers:
   - label: Monarch
     title: Monarch Phenotype Page
     url: http://monarchinitiative.org/phenotype/MP:0000001
+usages:
```
Comment (matentzn) [inline ontology/mp.md:46]: @sbello can you digg out other usages, like IMPC, in the same way as we did this one here? The more the better!

Comment (sbello) [reply ontology/mp.md:46]: Now that the formatting issue is fixed I'll add some additional usages

**Comment by matentzn** [comment]
Closing in favour of https://github.com/OBOFoundry/OBOFoundry.github.io/pull/1593

**Pull request closed without merging**